packages/camera/camera_windows/windows/capture_controller.cpp

```diff
@@ -533,8 +535,6 @@ void CaptureControllerImpl::StopRecord() {
   // Check MF_CAPTURE_ENGINE_RECORD_STOPPED event handling for response
   // process.
   if (!record_handler_->StopRecord(capture_engine_.Get())) {
-    // Destroy record handler on error cases to make sure state is resetted.
-    record_handler_ = nullptr;
```

Comment:
`OnRecordStopped` does nothing if `record_handler_` is null, causing errors to be dropped. Also, `OnRecordStopped` cleans up the record handler properly.